sub/project_settings/{{ cookiecutter.__folder_name }}/backend/scripts/create_site.py

```diff
@@ -66,7 +71,10 @@ def asbool(s):
     portal_setup.runAllImportStepsFromProfile("profile-{{ cookiecutter.python_package_name }}:default")
     transaction.commit()
 
+{% if cookiecutter.__feature_headless == "1" %}
     if EXAMPLE_CONTENT:
         portal_setup.runAllImportStepsFromProfile("profile-{{ cookiecutter.python_package_name }}:initial")
         transaction.commit()
+{%- endif %}
```

Comment:
Why are you removing this for the Classic UI project? The backend add-on still defines an initial profile, but without this it is never used.

Reply:
The initial-profile imports content from `setuphandlers/examplecontent/content` which is Volto-specific and fails in Classic because there are Images inside a Document (Documents are non-folderish in Classic). 
I would prefer to use the distributions from `plone.volto` and `plone.classicui` instead of maintaining another set of example-content here.

Reply:
@pbauer Okay, got it.

Reusing the example content from the distributions makes sense as a starting point, but won't many projects want to edit the content and export it to create their own customized example content? So we should probably still create an (empty) examplecontent folder and set up the code to load it, even if it's empty to begin with.